mobile/lib/infrastructure/repositories/remote_album.repository.dart

```diff
@@ -10,26 +10,48 @@ class DriftRemoteAlbumRepository extends DriftDatabaseRepository {
   const DriftRemoteAlbumRepository(this._db) : super(_db);
 
   Future<List<Album>> getAll({Set<SortRemoteAlbumsBy> sortBy = const {}}) {
-    final query = _db.remoteAlbumEntity.select();
+    final assetCount = _db.remoteAlbumAssetEntity.assetId.count();
+
+    final query = _db.remoteAlbumEntity.select().join([
+      leftOuterJoin(
+        _db.remoteAlbumAssetEntity,
+        _db.remoteAlbumAssetEntity.albumId.equalsExp(_db.remoteAlbumEntity.id),
+        useColumns: false,
+      ),
+      leftOuterJoin(
+        _db.userEntity,
+        _db.userEntity.id.equalsExp(_db.remoteAlbumEntity.ownerId),
+      ),
+    ]);
+    query
+      ..addColumns([assetCount])
+      ..groupBy([_db.remoteAlbumEntity.id]);
 
     if (sortBy.isNotEmpty) {
-      final orderings = <OrderClauseGenerator<$RemoteAlbumEntityTable>>[];
+      final orderings = <OrderingTerm>[];
       for (final sort in sortBy) {
         orderings.add(
           switch (sort) {
-            SortRemoteAlbumsBy.id => (row) => OrderingTerm.asc(row.id),
+            SortRemoteAlbumsBy.id => OrderingTerm.asc(_db.remoteAlbumEntity.id),
           },
         );
       }
       query.orderBy(orderings);
     }
 
-    return query.map((row) => row.toDto()).get();
+    return query
+        .map(
+          (row) => row.readTable(_db.remoteAlbumEntity).toDto(
+                assetCount: row.read(assetCount) ?? 0,
+                ownerName: row.readTableOrNull(_db.userEntity)?.name,
```

Comment:
```suggestion
                ownerName: row.readTable(_db.userEntity).name,
```

And you can make the `ownerName` argument of the `toDto` extension a required one